beacon_node/network/src/attestation_service/mod.rs

```diff
@@ -47,12 +47,13 @@ pub enum AttServiceMessage {
     /// Remove the `SubnetId` from the ENR bitfield.
     EnrRemove(SubnetId),
     /// Discover peers for a particular subnet.
-    DiscoverPeers(SubnetId),
+    /// The associated slot indicates the slot we need the discovered peer until.
+    DiscoverPeers(ExactSubnet),
 }
 
 /// A particular subnet at a given slot.
-#[derive(PartialEq, Eq, Hash, Clone)]
-struct ExactSubnet {
+#[derive(PartialEq, Eq, Hash, Clone, Copy, Debug)]
```

Comment:
why add copy here?

Reply:
The tests weren't working once I updated DiscoverPeers(SubnetId) to DiscoverPeers(ExactSubnet), so I added it.  I think just borrowing in the tests assertions also would have worked.  But as I update the event to DiscoverPeers{SubnetId, Instant}, I'll remove it.